CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
-### 🔄 Changed
+## StreamChatUI
+### 🐞 Fixed
+- Fix overriding of actual current composer link preview [#3025](https://github.com/GetStream/stream-chat-swift/pull/3025)
```

Comment:
Maybe change the wording a bit here to be more clear? ("actual current" is not entirely clear imo)

Reply:
WDYT of this one:
- Fix composer link preview overridden by previous enrichment

Reply:
much better 👍